shared/tracker/proofs.render.desktop.js

```diff
@@ -102,7 +122,9 @@ ProofsRender.propTypes = {
   username: React.PropTypes.string.isRequired
 }
 
-const styles = {
+const styles = (flags.tracker2 ? styles2 : styles1)
```

Comment:
seems like it might get confusing if you forget to pass in style somewhere and this one takes its place instead of erroring.